build.cake

```diff
@@ -2,6 +2,8 @@
 #tool nuget:?package=GitVersion.CommandLine
 #addin "Cake.Incubator"
 
+using System.Text.RegularExpressions;
```

Comment:
Why is this necessary ?

Reply:
I had a regular expression at one point but it's gone now.